apps/snowflake/data/index.html

```diff
@@ -0,0 +1,130 @@
+                </defs>
+            </svg>                                                  
+            <div class="app-details">
+                <span class="app-status success">&#9679; Running</span>
+                <h1 class="app-name">Tor Snowflake Proxy</h1>
+            </div>
+        </div>
+        <p class="text-muted">
+            Thank you for installing Tor Snowflake Proxy on your Umbrel to help defeat Tor censorship around the world.
```

Comment:
Isn't this helping to defeat censorship in general? Or is it only 'Tor censorship' ?

e.g. Thank you for installing the Tor Snowflake Proxy on your Umbrel to help defeat censorship around the world.

Reply:
Snowflake specifically is built to help people access the Tor network in countries where Tor is blocked. So while Tor helps defeat _censorship_, Snowflake helps defeat _Tor censorship_. Maybe @lukechilds can confirm?